src/transformation-components/SelectAttributes.tsx

```diff
@@ -62,26 +67,25 @@ export function SelectAttributes({
   return (
     <>
       <p>Table to Select Attributes From</p>
-      <select
-        id="inputDataContext"
+      <CodapFlowSelect
         onChange={inputChange}
-        defaultValue="default"
-      >
-        <option disabled value="default">
-          Select a Data Context
-        </option>
-        {dataContexts.map((dataContext) => (
-          <option key={dataContext.name} value={dataContext.name}>
-            {dataContext.title} ({dataContext.name})
-          </option>
-        ))}
-      </select>
+        options={dataContexts.map((dataContext) => ({
+          value: dataContext.name,
+          title: dataContext.title,
+        }))}
+        value={inputDataCtxt}
+        defaultValue="Select a Data Context"
+      />
 
       <p>Attributes to Include in Output (1 per line)</p>
-      <textarea onChange={attributesChange}></textarea>
+      <CodapFlowTextArea onChange={attributesChange} value={attributes} />
 
       <br />
-      <button onClick={() => transform()}>Select attributes!</button>
+      <TransformationSubmitButtons
+        onCreate={() => transform()}
+        onUpdate={() => transform()}
+        updateDisabled={true}
+      />
```

Comment:
Most of the transformation buttons aren't hooked up yet since most components don't have table update functionality. Eventually they will look like
```js
<TransformationSubmitButtons
onCreate={() => transform(false)}
onUpdate={() => transform(true)}
updateDisabled={!lastContextName} 
/>
```